Backpack/src/main/java/net/skyscanner/backpack/calendar2/CalendarParams.kt

```diff
@@ -29,7 +29,9 @@ import org.threeten.bp.temporal.WeekFields
 data class CalendarParams(
   val range: ClosedRange<LocalDate>,
   val selectionMode: SelectionMode,
-  val cellInfo: Map<LocalDate, CellInfo> = emptyMap(),
+  val cellsInfo: Map<LocalDate, CellInfo> = emptyMap(),
+  val defaultCellInfo: CellInfo = CellInfo(),
+  val disabledDates: Set<LocalDate> = emptySet(),
```

Comment:
how come this is extracted?

Reply:
We use CellInfo as default as well now (for instance, to provide a dash for the cells without price). If disabling information will be available inside the info it'll be possible to disable all the cells in the calendar, which I believe not something we should "promote".

Therefore it was extracted.

Reply:
Maybe instead of a separate array having the `disabled` state be defined as part of `CellInfo` (but outside of the `status`?